apps/lockfile-explorer/src/start.ts

```diff
@@ -103,7 +105,17 @@ function startApp(debugMode: boolean): void {
 
   app.get('/api/lockfile', async (req: express.Request, res: express.Response) => {
     const pnpmLockfileText: string = await FileSystem.readFileAsync(appState.pnpmLockfileLocation);
-    const doc = yaml.load(pnpmLockfileText);
+    const doc = yaml.load(pnpmLockfileText) as Lockfile;
+    const { packages, lockfileVersion } = doc;
+    if (packages && lockfileVersion.toString().startsWith('6.')) {
```

Comment:
Is this fix really about supporting multiple versions  of the PNPM lockfile format?  That might be a better description for the change log.

Reply:
![image](https://github.com/microsoft/rushstack/assets/68707557/d3760604-0334-410d-8fd5-b2a79eeed17a)
I think the previous code already supports multiple versions of the PNPM lockfile format, although there are some issues.